### PR TITLE
Agt 25 home page

### PR DESCRIPTION
**Pull request by linjialiang1234**

### CODE REVIEW

File: .application-config.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
```
Comment (beltekylevi) [inline .application-config.json:1]: didn't we get rid of this file? 🤔

File: src/client/app/component/BusinessOverview/index.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import './style.scss';
+import SingleBusinessOverview from '../SingleBusinessOverview';
+
+class BusinessOverview extends React.Component {
+  render() {
+    let imagesDetails = [
```
Comment (beltekylevi) [inline src/client/app/component/BusinessOverview/index.js:7]: 🍬 put the static data into a separate file and use a little bit more precise placeholder text like lorem ipsum 😄

File: src/client/app/component/BusinessOverview/index.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        'ajflkdjalfjdaljfdlasjfljfaljfalfjlajfldafahdddd' +
+        'dddddddddddddrjgtkrejtkFdd',
+        businessKeywords: '#Nishi-nari Ward',
+        businessMore: 'http://www.baidu.com',
+      },
+    ];
+
+    let allBusiness = [];
+
+    imagesDetails.map(
```
Comment (beltekylevi) [inline src/client/app/component/BusinessOverview/index.js:88]: map returns a list, so you don't have to push into `allBusiness` list manually

File: src/client/app/component/BusinessOverview/index.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        allBusiness.push(
+          <SingleBusinessOverview itemInfo={item} />
+        );
+      }, this);
+
+    return (
+      <div id="business-overview-container">
+        {allBusiness}
+      </div>
+    );
```
Comment (beltekylevi) [inline src/client/app/component/BusinessOverview/index.js:100]: 🍬 nice formatting! 🙂

File: src/client/app/component/HomePage/index.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+import HomePageHeader from '../HomePageHeader';
+import HomePageContainer from '../HomePageContainer';
+import HomePageMap from '../HomePageMap';
+
+import './style.scss';
+
+class HomePage extends React.Component {
+  render() {
+    return (
+      <div className = "HomePage">
```
Comment (beltekylevi) [inline src/client/app/component/HomePage/index.js:11]: do you need to add spaces around the `=` sign?

File: src/client/app/component/HomePage/style.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
+$mapWidth: 45%;
+$containerWidth: 55%;
+$testHeight: 64.6875em;
+
+.HomePage {
+  width: 100%;
+}
+
+.Main{
```
Comment (beltekylevi) [inline src/client/app/component/HomePage/style.scss:9]: space after selector

File: src/client/app/component/HomePage/style.scss [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  height: $testHeight;
+}
+
+.HomePageContainer{
```
Comment (beltekylevi) [inline src/client/app/component/HomePage/style.scss:16]: space after selector

File: src/client/app/component/HomePage/style.scss [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+  order: 0;
+  flex-basis: $containerWidth;
+}
+
+.HomePageMap{
```
Comment (beltekylevi) [inline src/client/app/component/HomePage/style.scss:21]: space after selector

File: src/client/app/component/HomePageMap/index.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import './style.scss';
+
+class HomePageMap extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      businesses: [{lat: 22.528640199999998, lng: 113.94},
```
Comment (beltekylevi) [inline src/client/app/component/HomePageMap/index.js:9]: 🍬 put the static data into a separate file

File: .gitignore [hunk 10]
```diff
@@ -1,5 +1,6 @@
 .idea
 node_modules/
 dist/app
+.env
 *.log
-.application-config.json
\ No newline at end of file
+*.DS_Store
```
Comment (beltekylevi) [inline .gitignore:6]: `.DS_Store`

File: src/client/app/component/HomePageHeader/style.scss [hunk 11]
```diff
@@ -0,0 +1,28 @@
+.HomePageHeader{
```
Comment (beltekylevi) [inline src/client/app/component/HomePageHeader/style.scss:1]: space after every selector

File: src/client/app/component/NavigationBar/style.scss [hunk 12]
```diff
@@ -0,0 +1,45 @@
+$font-family: FSElliotPro-Bold;
+$border: 1px solid black;
+$red: #D93738;
+$yellow: #F8E71C;
+$white: #FFFFFF;
+
+#nav-container {
```
Comment (beltekylevi) [inline src/client/app/component/NavigationBar/style.scss:7]: avoid using ids

File: src/client/app/component/NavigationBar/style.scss [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+  width: 9.9375em;
+  height: 0.875em;
+  font-size: 0.85em;
+  padding: 1.4em 0em;
+  text-decoration: none;
+  text-align: center;
+  border-bottom: 1px solid yellow;
+}
+
+#comment-img {
```
Comment (beltekylevi) [inline src/client/app/component/NavigationBar/style.scss:39]: avoid using ids

**Review by beltekylevi — CHANGES_REQUESTED** [review]
let's go through everything in person as well

**Comment by Jerry-ZWL** [comment]
it was done already by another branch

**Pull request closed without merging**